mlflow/fastai.py

```diff
@@ -381,7 +381,11 @@ def load_model(model_uri):
 @experimental
 @autologging_integration(FLAVOR_NAME)
 def autolog(
-    log_models=True, disable=False, exclusive=False, disable_for_unsupported_versions=False
+    log_models=True,
+    disable=False,
+    exclusive=False,
+    disable_for_unsupported_versions=False,
+    silent=False,
```

Comment:
Add a `silent` param with docstring description to each framework-specific autologging function (e.g. `mlflow.fastai.autolog()`, `mlflow.sklearn.autolog()`). Also add the `silent` parameter to the universal `mlflow.autolog()` function.